retirement_api/fixtures/retiredata.json

```diff
@@ -1,6 +1,14 @@
3, \"age 64\": 1085, \"age 69\": 1567, \"age 68\": 1461, \"age 70\": 1673}, \"disability\": \"\", \"early retirement age\": \"\", \"params\": {\"prgf\": 2, \"lastYearEarn\": \"\", \"dobmon\": 4, \"dollars\": 1, \"retireyear\": 2024, \"yob\": 1958, \"lastEarn\": \"\", \"dobday\": 3, \"retiremonth\": 12, \"earnings\": 40000}, \"months_past_birthday\": 0, \"full retirement age\": \"66 and 8 months\", \"survivor benefits\": {\"spouse at full retirement age\": \"\", \"family maximum\": \"\", \"spouse caring for child\": \"\", \"child\": \"\"}}}, \"born_on_3rd_in_1959\": {\"note\": \"\", \"params_adjusted\": false, \"current_age\": 57, \"past_fra\": false, \"error\": \"\", \"data\": {\"benefits\": {\"age 63\": 1022, \"age 62\": 960, \"age 67\": 1366, \"age 66\": 1348, \"age 65\": 1183, \"age 64\": 1093, \"age 69\": 1582, \"age 68\": 1474, \"age 70\": 1690}, \"disability\": \"\", \"early retirement age\": \"\", \"params\": {\"prgf\": 2, \"lastYearEarn\": \"\", \"dobmon\": 4, \"dollars\": 1, \"retireyear\": 2026, \"yob\": 1959, \"lastEarn\": \"\", \"dobday\": 3, \"retiremonth\": 2, \"earnings\": 40000}, \"months_past_birthday\": 0, \"full retirement age\": \"66 and 10 months\", \"survivor benefits\": {\"spouse at full retirement age\": \"\", \"family maximum\": \"\", \"spouse caring for child\": \"\", \"child\": \"\"}}}, \"born_on_3rd_in_1970\": {\"note\": \"\", \"params_adjusted\": false, \"current_age\": 46, \"past_fra\": false, \"error\": \"\", \"data\": {\"benefits\": {\"age 63\": 1107, \"age 62\": 1039, \"age 67\": 1476, \"age 66\": 1378, \"age 65\": 1279, \"age 64\": 1181, \"age 69\": 1712, \"age 68\": 1594, \"age 70\": 1830}, \"disability\": \"\", \"early retirement age\": \"\", \"params\": {\"prgf\": 2, \"lastYearEarn\": \"\", \"dobmon\": 4, \"dollars\": 1, \"retireyear\": 2037, \"yob\": 1970, \"lastEarn\": \"\", \"dobday\": 3, \"retiremonth\": 4, \"earnings\": 40000}, \"months_past_birthday\": 0, \"full retirement age\": \"67\", \"survivor benefits\": {\"spouse at full retirement age\": \"\", \"family maximum\": \"\", \"spouse caring for child\": \"\", \"child\": \"\"}}}, \"born_on_3rd_in_1948\": {\"note\": \"Age 68 is past your full benefit claiming age.\", \"params_adjusted\": false, \"current_age\": 68, \"past_fra\": true, \"error\": \"\", \"data\": {\"benefits\": {\"age 63\": 0, \"age 62\": 0, \"age 67\": 0, \"age 66\": 0, \"age 65\": 0, \"age 64\": 0, \"age 69\": 1463.0, \"age 68\": 1355, \"age 70\": 1571.0}, \"disability\": \"You have reached full retirement age and are not eligible for disability benefits.\", \"early retirement age\": \"\", \"params\": {\"prgf\": 2, \"lastYearEarn\": \"\", \"dobmon\": 4, \"dollars\": 1, \"retireyear\": 2016, \"yob\": 1948, \"lastEarn\": \"\", \"dobday\": 3, \"retiremonth\": 4, \"earnings\": 40000}, \"months_past_birthday\": 0, \"full retirement age\": \"66\", \"survivor benefits\": {\"spouse at full retirement age\": \"\", \"family maximum\": \"\", \"spouse caring for child\": \"\", \"child\": \"\"}}}}", 
```

Comment:
May want to break this line up for readable (not highest priority though).
http://stackoverflow.com/questions/2058925/how-can-i-break-up-this-long-line-in-python